spiceypy/tests/test_wrapper.py

```diff
@@ -5701,6 +5703,8 @@ def test_scencd():
     sclkdp = spice.scencd(-32, sclkch)
     npt.assert_almost_equal(sclkdp, 985327950.0)
     assert sclkch == "2/20538:39:768"
+    mylist = 3 * [sclkch]
+    npt.assert_almost_equal(spice.scencd(-32, mylist), 3*[985327950.0])
```

Comment:
for these and other new unit tests, they should be independent test functions that include vectorized in the function name ie `test_scencd_vectorized`. see rest of test_wrapper.py for examples.